src/components/Components/ProjectDetailsPanelComponent/ProjectDetailsPanelComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import { SVGAtom, type IconNames } from "~/components/Atoms/SVGAtom/SVGAtom";
+
+export type ItemIcon = {
```

Comment:
It is recommended to use the type export to a minimum.
Over-exporting can result in duplication of the type name and dependence on the components that export the type.

My recommended method is to make an Atom with this type and use `React.ComponentProps<type of Atom>`.
This method is recommended because it can automatically change the types when the subcomponent type changes, the Atom name is simple to track, and it is difficult to duplicate due to the Convention Rule.

If there is a better way than the above method, please share your opinion and discuss with all developers in Slack.